src/wp-includes/script-loader.php

```diff
@@ -2770,6 +2770,11 @@ function _wp_normalize_relative_css_links( $css, $stylesheet_url ) {
 				continue;
 			}
 
+			// Skip if the URL is a HTML ID.
+			if ( str_starts_with( $src_result, '#' ) ) {
```

Comment:
This uses `str_starts_with()`, which was polyfilled in WordPress 5.9.

However, the checks for an absolute URL above use `0 === strpos()`.

I'm happy to change this from `str_starts_with()` to `0 === strpos()` for consistency, or a follow-up commit could change the checks above to use `str_starts_with()` for readability.

Reply:
Using `str_starts_with` looks good to me. Let's consider moving the above checks in a follow-up ticket 👍